### PR TITLE
Update styles.css

### PR DESCRIPTION
**Pull request by 95strat**
I Told You!

### CODE REVIEW

**Comment by 95strat** [comment]
:)

**Comment by ghost** [comment]
ok

File: styles.css [hunk 1]
```diff
@@ -13,5 +13,5 @@ p {
   display: block;
   width: 400px;
   margin: 50px auto;
-  font: 30px Monaco,"Courier New","DejaVu Sans Mono","Bitstream Vera Sans Mono",monospace;
+  font: 30px Monaco,"Freestyle Script","DejaVu Sans Mono","Bitstream Vera Sans Mono",monospace;
```
Comment (ghost) [inline styles.css:16]: looks good

**Comment by ghost** [comment]
test

File: styles.css [hunk 2]
```diff
@@ -13,5 +13,5 @@ p {
   display: block;
   width: 400px;
```
Comment (ghost) [inline styles.css:14]: ???

**Pull request closed without merging**